protocols/request-response/Cargo.toml

```diff
@@ -16,12 +16,21 @@ futures = "0.3.28"
 instant = "0.1.11"
 libp2p-core = { workspace = true }
 libp2p-swarm = { workspace = true }
+libp2p-swarm-derive = { workspace = true }
```

Comment:
```suggestion
libp2p-swarm = { workspace = true }
```

Reply:
Without `libp2p-swarm-derive = { workspace = true }`

$ cargo clean
$ cargo build

Throws the next error:
```
error: cannot find derive macro `NetworkBehaviour` in this scope
  --> protocols/request-response/src/json.rs:45:10
   |
45 | #[derive(NetworkBehaviour)]
   |          ^^^^^^^^^^^^^^^^
   |
   = help: consider importing one of these items:
           crate::NetworkBehaviour
           libp2p_swarm::NetworkBehaviour
```

Reply:
Without `libp2p-swarm-derive = { workspace = true }`

$ cargo clean
$ cargo build

Throws the next error:
```
error: cannot find derive macro `NetworkBehaviour` in this scope
  --> protocols/request-response/src/json.rs:45:10
   |
45 | #[derive(NetworkBehaviour)]
   |          ^^^^^^^^^^^^^^^^
   |
   = help: consider importing one of these items:
           crate::NetworkBehaviour
           libp2p_swarm::NetworkBehaviour
```